server/Controllers/RecipeController.cs

```diff
@@ -31,5 +31,23 @@ public async Task<IActionResult> GetDetails(int id)
         return NotFound();
       }
     }
+
+    [HttpGet]
+    [Route("all_recipes/{searchQuery}")]
```

Comment:
`search` route fits better. Instead of making use of URL parameters, use a POST body instead.